### ISSUE TITLE
error LNK2001 unresolved external symbol _Py_NoneStruct

### ISSUE THREAD

**Issue by githubzhanglei** [Microsoft/caffe#72]
I follow the instructions to build windows caffe. All the others projects are bulid successfully, however,  the errors reported when I build the pycaffe project. The errors are as follows:
error LNK2001 unresolved external symbol  _Py_NoneStruct
error LNK2001 unresolved external symbol  PyInt_Type
error LNK2001 unresolved external symbol  PyBool_Type
error LNK2001 unresolved external symbol  PyFloat_Type
error LNK2001 unresolved external symbol  PyString_Type
error LNK2001 unresolved external symbol  PyTuple_Type
error LNK2001 unresolved external symbol  PyCObject_Type
error LNK2001 unresolved external symbol  PySlice_Type
error LNK2001 unresolved external symbol  PyExc_AttributeError
error LNK2001 unresolved external symbol  PyExc_ImportError
error LNK2001 unresolved external symbol  PyExc_IndexError
error LNK2001 unresolved external symbol  PyExc_RuntimeError
error LNK2001 unresolved external symbol  PyExc_TypeError

Anyone knows how to fix the problem?
Thanks.

**Comment by githubzhanglei** [comment]
fix the errors by adding all the libboost_python libs in the "..\NugetPackages\boost_python2.7-vc120.1.59.0.0\lib\native\address-model-64\lib\" folders to the pycaffe project settings

**Comment by mnidza** [comment]
It's strange that you had to do that, because that folder is supposed to be already included in library search path via `boost_python-vc120.targets`, which is imported from `pycaffe.vcxproj` (line 85).

**Comment by githubzhanglei** [comment]
Yes, you are right. I try to delete those libs which I added before, there is no error reported. I am trying to reproduce those errors now, and figuring out the problem.
